### PR TITLE
 Split "Failed Verify" into more useful audit entries

### LINKED ISSUE #113: "Failed Verify" as a CRL audit term is not explanatory
Most of the "Failed Verify" entries are actually more like "Obtained Old CRL", so we should special-case that event and leave "Failed Verify" to actual signature errors.

### PR DESCRIPTION
**Pull request by jcjones**
Fixes #113. Most of the "Failed Verify" entries are actually more like
 "Obtained Old CRL", so this special-cases those out into "Older Than Previous"
 and also covers Expiration more clearly, since those were not tracked before.

### CODE REVIEW

**Comment by jcjones** [comment]
In `dev` as 3cd1842d0dc81519db4c564d66c1b46d5aad578c

**Comment by jcjones** [comment]
The "Older Than Previous" entries don't provide DNS information, which is unfortunate. Should fix that.

**Comment by jcjones** [comment]
Hmmm, that _should_ be working, from what I see in the code.

File: go/cmd/aggregate-crls/crl-auditor.go [hunk 1]
```diff
@@ -98,6 +101,23 @@ func (auditor *CrlAuditor) FailedVerifyUrl(issuer storage.Issuer, crlUrl *url.UR
 	})
 }
 
+func (auditor *CrlAuditor) FailedOlderthanPrevious(issuer storage.Issuer, crlUrl *url.URL, dlAuditor *DownloadAuditor, previous time.Time, this time.Time) {
```
Comment (kjacobs-moz) [inline go/cmd/aggregate-crls/crl-auditor.go:104]: s/Olderthan/OlderThan 
(same with the test function).

Comment (jcjones) [reply go/cmd/aggregate-crls/crl-auditor.go]: Thanks!

**Pull request merged**